src/compiler/file_uploader/on_before_exit.ts

```diff
@@ -1,12 +1,9 @@
 import { Dest, Src } from '.';
 import { getListOfIncompleteFiles } from './incomplete_files';
-import { getOngoingHashingJobs, OngoingHashingJob } from './ongoing_hashes';
 import { UploaderActor } from './uploader_actor';
 
 export function onBeforeExit(paths: [Src, Dest][], actor: UploaderActor): void {
-    let hashingComplete = false;
     let cleanUpComplete = false;
```

Comment:
Why does this cleanup process need to be a separate process? Can it not be done somehow within the same update calls as the chunk uploading?

Reply:
We're in the middle of accomplishing this

Reply:
https://github.com/demergent-labs/azle/issues/1995

Reply:
The issue linked in the previous reply doesn't seem correct